app/views/posts/_post-card.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="post-card border border-gray-300 rounded-md p-5 mb-5 bg-gray-100">
+    <h4 class="font-semibold text-lg mb-5"><%= post.title %> </h4>
+    <p>
+        <%= post.text %>
+    </p>
+
+    <p class="text-right text-sm mt-3">Comments: 10 Likes: 4</p>
```

Comment:
Kindly make sure that you display the number of comments and likes based on the comments_counter and likes_counter. Here you are displaying fixed values for comments and likes.

Reply:
Hello @KDlamini, thank you for the reminder. I forgot about that section. I will implement the changes